### DISCUSSION TITLE
[Question]:

### DISCUSSION THREAD

**Discussion by vislamu** [danny-avila/LibreChat#3720] [Q&A]
### What is your question?

about how to count token when use Dalle-3 to generate images

### More Details

If use Dalle-3 to generate images, how did the token usage? Where can I set it or make some modify?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vislamu** [comment]
I enabled token statistics and used plugins mode to select the GPT-4O model and Dalle-3 to generate images. It seems that the tokens(balance) consumed do not include the image generation part. Did I not set it correctly somewhere?

**Comment by danny-avila** [comment]
Duplicate: https://github.com/danny-avila/LibreChat/discussions/1479
